README.md

```diff
@@ -28,6 +37,73 @@ Each input signal triggers a pin read. The boolean `pin` value is added to the s
 ```
 {
   'input_attr': 'I was already here',
-  'value': True,
+  'value': True
+}
+```
+
+GPIOWrite
+========
+
+Write to the specified gpio pin.
+
+Properties
+----------
+- **pin** (int): pin to write to
+- **value** (bool): bool value to write to `pin`
+
+Dependencies
+------------
+None
+
+Commands
+--------
+None
+
+Input
+------
+Any list of signals.
+
+Output
+------
+
+Each input signal triggers a pin write. The boolean `pin` value is added to the signal.
+
+```
+{
+  'input_attr': 'I was already here',
+  'value': 'value_property'
+}
+```
+
+GPIOInterrupt
+==============
+Send an interrupt to the specified gpio pin.
```

Comment:
It's actually the opposite. An "interrupt" signal is notified out of the block whenever a pin changes it's boolean state.

https://en.wikipedia.org/wiki/Interrupt